src/ddsrt/include/dds/ddsrt/md5.h

```diff
@@ -60,7 +60,9 @@
 #include <stddef.h>
 #include "dds/export.h"
 
-/*
+/** @file md5.h
+ * The message digest algorithm computes a 128-bit hash from a message.
```

Comment:
I would say "the MD5 hash (a 128-bit hash, still suitable for detecting unintentional corruption, but no longer to be used where a cryptographic hash is needed)". The parenthesised bit is for those who are a bit behind.

Reply:
I know some people might have a different opinion, but my rule of thumb is that if something is an abbreviation or acronym, there should be at least in one place a short explanation (in a report that would be where it is first mentioned, in code I think the header file of that component is a suitable place).

Reply:
It is a good rule of thumb, but "the message digest algorithm" is actually leaving undefined *which* digest, and everyone refers to it simply as "MD5" so expanding it doesn't really help anyone.

And I dare say it was quite obvious from the function name what it would do anyway ...